packages/katello/katello/katello-backup-rotate-tar.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+## For this script it's advisable to use a shell, such as Bash,
+## that supports a TAR_FD value greater than 9.
+
+# This script taken from the tar info doc
+
+name=`expr ${TAR_ARCHIVE} : '\(.*\)\..*'`
+volnum=`printf "%03d" ${TAR_VOLUME}`
+filename="${name:-$TAR_ARCHIVE}.part${volnum:-$TAR_VOLUME}"
+case ${TAR_SUBCOMMAND} in
+-c)       ;;
+-d|-x|-t) test -r ${name:-$TAR_ARCHIVE}.part${volnum:-$TAR_VOLUME} || exit 1
```

Comment:
This can be replaced with `filename` too right?

Reply:
good call, fixed